config_full.json.example

```diff
@@ -20,9 +20,14 @@
         "0":  0.04
     },
     "stoploss": -0.10,
-    "unfilledtimeout": {
-        "buy": 10,
-        "sell": 30
+    "unfilled_timeout": {
+        "buy": {
+            "after": 10,
+            "even_if_buy_signal_valid": true
+        },
+        "sell": {
+            "after": 30
+        }
```

Comment:
I don't particularly like the nested dicts here (mainly because it'd break existing configurations).

How about the following:


``` json
    "unfilled_timeout": {
     "buy": 10,
     "buy_timeout_with_buy_signal": 60,
     "sell": 30
    },	  
```

Notice that it'll use a number, not a boolean - so you can simply use a "longer" timeout in this case (something discussed above).

Implementation wise, this should fall back to "buy" if not set - keeping the "original" behaviour.